### PR TITLE
Guion screen

### PR DESCRIPTION
**Pull request by leomendezg**
I fixed some things and I added the camera capture feature and Gallery feature.

### CODE REVIEW

File: app/src/main/java/com/globant/scriptsapadea/ui/activities/MainActivity.java [hunk 1]
```diff
@@ -62,6 +65,16 @@ public void onAboutClicked(View view) {
         }
     }
 
+
+    public void onCreateNewGuion(View view){
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/activities/MainActivity.java]: "Guion" ? use English

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/PictureFragment.java [hunk 2]
```diff
@@ -0,0 +1,102 @@
...
+
+    @Override
+    public void onActivityResult(int requestCode, int resultCode, Intent data) {
+        super.onActivityResult(requestCode, resultCode, data);
+
+        Log.d(PictureFragment.class.toString(),requestCode + "");
+    }
+
+    private void pickPhotoFromGallery() {
+
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/PictureFragment.java:85]: Remove empty space.

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/PictureFragment.java [hunk 3]
```diff
@@ -0,0 +1,102 @@
...
+        Intent galleryIntent = new Intent(Intent.ACTION_GET_CONTENT);
+        galleryIntent.setType("image/*");
+        startActivityForResult(galleryIntent,GALLERY);
+    }
+
+    private void takePhotoFromCamera(){
+
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/PictureFragment.java:92]: Remove empty space.

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+import roboguice.inject.InjectView;
+
+/**
+ * Created by leonel.mendez on 5/8/2015.
+ */
+public class ScreenplayFragment extends BaseFragment {
+
+
+
+
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java:29]: Why so many spaces ?

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java [hunk 5]
```diff
@@ -0,0 +1,78 @@
...
+    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
+        return inflater.inflate(R.layout.fragment_guion,container,false);
+    }
+
+    @Override
+    public void onViewCreated(View view, Bundle savedInstanceState) {
+        super.onViewCreated(view, savedInstanceState);
+
+
+
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java]: Remove empty space.

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java [hunk 6]
```diff
@@ -0,0 +1,78 @@
...
+        EditText screenplayName = (EditText)view.findViewById(R.id.screenplay_name);
+        Button nextButton = (Button)view.findViewById(R.id.next_button);
+
+        showNextButton(screenplayName,nextButton);
+        nextButton.setOnClickListener(new View.OnClickListener() {
+            @Override
+            public void onClick(View v) {
+
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java]: Remove empty space.

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java [hunk 7]
```diff
@@ -0,0 +1,78 @@
...
+                getActivity().getSupportFragmentManager().beginTransaction().replace(R.id.fragment_container,new PictureFragment()).addToBackStack("picture_fragment").commit();
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java]: TODO use the navigator next time.

File: captures/SystemInfo_2015.05.25_16.03.30.txt [hunk 8]
```diff
@@ -0,0 +1,89 @@
+TASK com.globant.scriptsapadea id=1365
```
Comment (nicolaquartieri) [inline captures/SystemInfo_2015.05.25_16.03.30.txt:1]: What is this file ?

File: app/src/main/res/layout/activity_main.xml [hunk 9]
```diff
@@ -9,6 +9,14 @@ android:paddingBottom="@dimen/activity_vertical_margin"
 android:orientation="vertical"
 tools:context=".MainActivity">
 
+    <ScrollView
+        android:layout_width="match_parent"
+        android:layout_height="match_parent">
+    <LinearLayout
```
Comment (nicolaquartieri) [inline app/src/main/res/layout/activity_main.xml:15]: Please use indentation.

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java [hunk 10]
```diff
@@ -0,0 +1,78 @@
...
+                nextButton.setVisibility(View.VISIBLE);
+                if (s.toString().length() == 0) {
+                    nextButton.setVisibility(View.GONE);
+                }
+
+            }
+        });
+    }
+
+
```
Comment (nicolaquartieri) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java:86]: Remove empty spaces.

File: app/src/main/java/com/globant/scriptsapadea/ui/activities/MainActivity.java (hunk 1 above)
Comment (leomendezg) [inline app/src/main/java/com/globant/scriptsapadea/ui/activities/MainActivity.java]: Sorry I missed this line!

File: app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java (hunk 7 above)
Comment (leomendezg) [inline app/src/main/java/com/globant/scriptsapadea/ui/fragments/ScreenplayFragment.java]: I am going to add the navigator into the code

File: captures/SystemInfo_2015.05.25_16.03.30.txt (hunk 8 above)
Comment (leomendezg) [inline captures/SystemInfo_2015.05.25_16.03.30.txt:1]: I do not what file is, i am going to check it.

**Pull request closed without merging**